tiledb/api/src/lib.rs

```diff
@@ -79,3 +78,19 @@ pub trait Factory<'ctx> {
 
     fn create(&self, context: &'ctx context::Context) -> Result<Self::Item>;
 }
+
+mod private {
+    // the "sealed trait" pattern prevents downstream crates from implementing a trait
+    // if you make `Sealed` a super-trait. This is important for something like `PhysicalType`.
+    pub trait Sealed {}
```

Comment:
Yeah that's totally fair, I'll clarify the motivation.  Yep definitely want to avoid surprises if someone makes a `u256` or something and is like "tiledb should be able to use thhat!"

Reply:
Excellent. Thanks!